boards/arm/efr32mg_sltb004a/doc/efr32mg_sltb004a.rst

```diff
@@ -0,0 +1,202 @@
+.. _efr32mg_sltb004a:
+
+EFR32MG-SLTB004A
+###############
```

Comment:
The title underline must be at least as long as the title above it, so add one more ``#``

Reply:
Addressed.